instrumentation/opentelemetry-instrumentation-sqlalchemy/src/opentelemetry/instrumentation/sqlalchemy/engine.py

```diff
@@ -111,6 +114,11 @@ def __init__(
         listen(engine, "after_cursor_execute", _after_cur_exec)
         listen(engine, "handle_error", _handle_error)
 
+    def remove_event_listeners(self):
```

Comment:
its a private method should start with `_` prefix

Reply:
It's not a private method, I need to use this function from sqlalchemy